Optimize abstracts of Mng AKs (CQA) (#4960)
theforeman/foreman-documentation: guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc

= Enabling and disabling repositories on activation key

[role="_abstract"]

> Override repository set status on an activation key to control which repositories are enabled when hosts register with the activation key.

Reviewer: **I am not 100% sure about this**: IMO this also affects existing hosts when they run any package manager command or `subscription-manager`. -> Not only new hosts are affected, but _all_ hosts should be affected. Needs tech ACK.

My suggestion tries to rephrase "when hosts register" to also include hosts already registered with that AK:

```suggestion
Override repository set status on an activation key to control which repositories are enabled through the activation key on registered hosts.
```

Author: @jeremylenz Can you, please, confirm that AKs only enable/disable repositories during host registration/provisioning?

Reviewer 2: **Activation keys only affect hosts at registration time.** There are no exceptions to this.

The previous text was correct.

> Override repository set status on an activation key in the {ProjectWebUI} to control which repositories are enabled when hosts register with the key.

Author: ```suggestion
Override repository set status on an activation key to control which repositories are enabled when hosts register with the activation key.
```
